libraries/Microsoft.Bot.Builder/Teams/TeamsInfo.cs

```diff
@@ -32,7 +32,7 @@ public static class TeamsInfo
         /// <remarks>InvalidOperationException will be thrown if meetingId, participantId or tenantId have not been
         /// provided, and also cannot be retrieved from turnContext.Activity.</remarks>
         /// <returns>Team participant channel account.</returns>
-        public static async Task<TeamsParticipantChannelAccount> GetMeetingParticipantAsync(ITurnContext turnContext, string meetingId = null, string participantId = null, string tenantId = null, CancellationToken cancellationToken = default)
+        public static async Task<TeamsMeetingParticipant> GetMeetingParticipantAsync(ITurnContext turnContext, string meetingId = null, string participantId = null, string tenantId = null, CancellationToken cancellationToken = default)
```

Comment:
But isn't this a _source_ breaking change? (i.e. not just binary)

(Perhaps a type relationship between TeamsMeetingParticipant and TeamsParticipantChannelAccount would help? Or keep the old name if it's just additional properties?)

Reply:
TeamsMeetingParticipant now contains a TeamsChannelAccount field ('user').

We haven't released this api yet, but are about to.

Reply:
Awesome, so we hadn't released the previous GetMeetingParticipantAsync yet, that's good news, I hadn't realized that!